test/Altinn.App.Api.Tests/Altinn.App.Api.Tests.csproj

```diff
@@ -30,6 +30,10 @@
     <ProjectReference Include="..\Altinn.App.Common.Tests\Altinn.App.Common.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Altinn.App.Core.Tests" />
```

Comment:
Should Core tests depend on Api tests? If we have code that needs to be shared between both, maybe that's an argument for putting it in Altinn.App.Common.Tests

Reply:
I have no optinion on this. I just wanted access to some stuff across that boundary :)

Reply:
I'm putting this in the too-hard basket for now: https://github.com/Altinn/app-lib-dotnet/issues/916